assets/src/scss/new-identity/style.scss

```diff
@@ -1,12 +1,16 @@
 @import "../base/tokens";
+@import "../base/variables";
+@import "../base/mixins";
 
 @mixin shared-link-styles {
+  --link--height: 26px;
+  --transition--duration: 0.3s;
+
   border-bottom: 2px solid var(--gp-green-500);
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+  transition: var(--transition--duration) cubic-bezier(0.5, 0, 0, 1);
 
-  &:hover,
-  &:focus {
-    box-shadow: inset 0 calc(-22px) var(--gp-green-100);
+  &:hover {
```

Comment:
The `:active` selector should be added here! Right now it still has the extra underline.

<img width="246" alt="Screenshot 2023-05-03 at 08 52 21" src="https://user-images.githubusercontent.com/6949075/235849150-f92927d7-e04f-4e67-8c8a-7b1e07a4bd37.png">

I've also just noticed that right now on focus there is a [border-radius](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_typography.scss#L106) that should be applied on `:focus-visible` instead, but that could be done in a separate PR!

<img width="254" alt="Screenshot 2023-05-03 at 08 50 23" src="https://user-images.githubusercontent.com/6949075/235849080-2a0b5bfa-7bca-4c1d-bbed-48f9171352b8.png">

Reply:
I remember I pushed this change 🤔 . I'll update it soon !